apps/storefront/src/shared/service/b2b/graphql/login.ts

```diff
@@ -0,0 +1,30 @@
+import { convertArrayToGraphql } from '@/utils';
+
+import B3Request from '../../request/b3Fetch';
+
+interface ApiTokenConfig {
+  storeHash: string;
+  channel_id: number;
+  expires_at: number;
+  allowed_cors_origins: string[];
+}
+
+const storeFrontToken = (data: Partial<ApiTokenConfig>) => `mutation {
```

Comment:
Are all the properties in ApiTokenConfig optional when calling the `storeFrontToken` method?